helmfile/ibft2/charts/besu-node/templates/config-toml-configmap.yaml

```diff
@@ -2,29 +2,30 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: {{ include "besu-node.fullname" . }}-config
+  name: {{ include "besu-node.fullname" . }}-config-toml
```

Comment:
Not a comment on this line specifically, but the entire file: The complexity of breaking this out into values makes me wonder if we shouldn't just use the include file trick that you used above for the `prometheus.yml` file